src/schema/schema.graphql

```diff
@@ -2808,6 +2814,7 @@ type Mutation {
 		url: String
 		view: FolderView
 	): Folder
+	createIdentity(name: String!, attrs: IdentityAttrsInput!): Identities
```

Comment:
attrs are required in this line and optional in https://github.com/Zimbra/zm-api-js-client/pull/332/files#diff-28ea553bc23a82875eeae07c5452be22R2121

Reply:
same issue present in modifyIdentity

Reply:
I think we can directly use `CreateIdentityInput` here

Reply:
@silentsakky I am updating this line to be optional instead of required. Should I update it for modifyIdentity as well?

@yashrajpchavda looking at the way the other mutations are written, I don't believe this is correct. `CreateIdentityInput` is used in schema.ts.

Reply:
yes please update for modifyIdentity as well